### DISCUSSION TITLE
Demo website is flickering on my machine

### DISCUSSION THREAD

**Discussion by Yazington** [pmndrs/zustand#1418] [Q&A]
## Summary

Demo website is flickering on my machine 

## Check List

Please do not ask questions in issues.

- [x ] I understand this is not a question.

Please fill this template if you're filling an issue regarding TypeScript.

- [ ] I've added a link to a [typescript playground](https://www.typescriptlang.org/play) or [codesandbox](https://codesandbox.io) with a minimal reproduction. Or some other way of reproduction if these two don't suffice.
- [ ] I've read the [typescript guide](https://github.com/pmndrs/zustand/blob/main/docs/guides/typescript.md), in particular that `create` is to be used as `create<T>()(...)` and not `create<T>(...)`.

**Comment by dai-shi** [comment]
What is the URL?

**Reply by Yazington** [reply]
[zustand-demo.pmnd.rs/](https://zustand-demo.pmnd.rs/)

It was flickering on a laptop and its not as fast as my desktop. Desktop is faster.

**Reply by dai-shi** [reply]
Thanks. I think it's how three.js or something works. I'm not expert on this. It's done by community contributors.
Let's see if someone can answer better: @stfnsr @rnike @DanielPower

**Comment by nettum** [comment]
Same her, but only in chrome. Running `Ubuntu 22.04.1 LTS`, `Dell XPS 15 9510`. Using the integrated Intel UHD drivers :sweat_smile: 
Flickering: `Google Chrome 107.0.5304.110`, no flickering: `Mozilla Firefox 107.0`

**Reply by dai-shi** [reply]
The source code is here: https://github.com/pmndrs/zustand/tree/main/examples/demo
Feel free to investigate further.
